src/registration/LabelledTextInput.tsx

```diff
@@ -0,0 +1,32 @@
+import { useUniqueId } from '@terra-ui-packages/components';
+import React from 'react';
+import { TextInput } from 'src/components/input';
+import { FormLabel } from 'src/libs/forms';
+
+interface LabelledTextInputProps {
+  disabled?: boolean;
+  required?: boolean;
+  value: string;
+  onChange: (value: string) => void;
+  labelStyle?: object;
+  inputStyle?: object;
```

Comment:
Use the CSSProperties type (imported from React) for style props.

```suggestion
  labelStyle?: CSSProperties;
  inputStyle?: CSSProperties;
```